src/main/java/org/example/P06_MultiplicationTable.java

```diff
@@ -3,10 +3,10 @@
 
 public class P06_MultiplicationTable {
     public static void main(String[] args) {
-        Scanner number = new Scanner(System.in);
+        Scanner inputNum = new Scanner(System.in);
```

Comment:
Scanner can input any character; it does not have to be a number, so naming a scanner inputNum has no sense. Find better name.